src/lib/strings.js

```diff
@@ -0,0 +1,32 @@
+function sayHello(string) {
+  return `Hello, ${string}!`;
+}
+
+function uppercase(abcdefghi) {
+  return abcdefghi.toUpperCase();
+}
+
+function lowercase(ABCDEFGHI) {
+  return ABCDEFGHI.toLowerCase();
+}
```

Comment:
These variable names are not great! You should name them based on whwt they represent, in this case a string:
```
function uppercase(string) {
  return string.toUpperCase();
}

function lowercase(string) {
  return string.toLowerCase();
}
```